### ISSUE TITLE
Quick fixes not showing up

### ISSUE THREAD

**Issue by saurabhk-sap-tech** [fidley/ABAPQuickFix#48]
Hi,

I've installed the extension, but cannot seem to figure out how to enable it. Ctrl + 1 does not seem to show any of the supported actions/quick fixes mentioned in the README. Am I missing some pre-requisite?
![image](https://user-images.githubusercontent.com/19816877/204904522-0d758159-fdc7-4184-bcb9-b4eb96fa9767.png)
![image](https://user-images.githubusercontent.com/19816877/204907529-5ceb0824-66fb-40f2-9240-5222154a317f.png)
![image](https://user-images.githubusercontent.com/19816877/204907546-6e3ac1fa-13b0-421f-b36f-b63ebd989c78.png)

**Comment by fidley** [comment]
Try not to select the EQ, just put cursor on it and call quick fix.

**Comment by saurabhk-sap-tech** [comment]
It had nothing to do with this plugin. I had installed the plugin using the eclipse marketplace. I could see it under the **Installed** tab of the marketplace. But there was no entry in preferences for this plugin. So I decided to re-install it. Uninstalled it using the marketplace and eclipse prompted me for a restart. Restarted eclipse, but I could still see this plugin under the Installed tab of the marketplace. But for some odd reason, the plugin is now working(after I uninstalled it [🤷](https://emojipedia.org/person-shrugging/)) and there's also an entry for it in the preferences section. So, seems to me it was an eclipse thing. Anyways, I got it to work, so closing this.
